### PR TITLE
Add manifest to install Archi

### PR DESCRIPTION
**Pull request by FilipDeVos**
Archi is an archimate enterprise architecture tool that can be
downloaded from www.archimatetool.com.

### CODE REVIEW

File: archi.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            "64bit": {
+                "url": "https://www.archimatetool.com/downloads/root$cleanVersion/Archi-Win64-$version.zip"
+            },
+            "32bit": {
+                "url": "https://www.archimatetool.com/downloads/root$cleanVersion/Archi-Win32-$version.zip"
+            }
+        },
+        "hash": {
+            "url": "$url.MD5",
+            "find": "MD5\\s+\\(.*\\)\\s+=\\s+([a-fA-F0-9]{32})"
```
Comment (Ash258) [inline archi.json]: Find could: `MD5 \\($basename\\) = ([A-Fa-f\\d]{32})`

File: archi.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        }
+    },
+    "shortcuts": [
+        [
+            "archi.exe",
+            "Archi"
+        ]
+    ],
+    "bin": [
```
Comment (Ash258) [inline archi.json]: Change to `"bin": "archi.exe"`

File: archi.json (hunk 1 above)
Comment (FilipDeVos) [reply archi.json]: I updated the regex as suggested.

File: archi.json (hunk 2 above)
Comment (FilipDeVos) [reply archi.json]: updated bin call as suggested

**Pull request merged**